ml-algorithms/build.gradle

```diff
@@ -48,6 +49,10 @@ dependencies {
     implementation("ai.djl.onnxruntime:onnxruntime-engine:0.21.0") {
         exclude group: "com.microsoft.onnxruntime", module: "onnxruntime"
     }
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}") {
```

Comment:
Same here, we can remove this as alerting tool moved to skills repo

Reply:
Done.